src/main/java/com/thebois/views/game/ViewableResource.java

```diff
@@ -0,0 +1,69 @@
+package com.thebois.views.game;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+
+import com.thebois.Pawntastic;
+import com.thebois.models.world.resources.ResourceType;
+import com.thebois.views.TextureUtils;
+
+/**
+ * Viewable Resource that correspond to TerrainType but has view related information.
```

Comment:
"Resource that corresponds to TerrainType"? An artifact of copy/paste?

Reply:
Whoops